crates/query-engine/metadata/src/metadata/native_queries.rs

```diff
@@ -10,27 +11,36 @@ use std::collections::BTreeMap;
 #[derive(Debug, Clone, PartialEq, Eq, Default)]
 pub struct NativeOperations {
     pub queries: NativeQueries,
-    pub mutations: NativeQueries,
+    pub mutations: NativeMutations,
 }
 
 impl NativeOperations {
     pub fn empty() -> Self {
         NativeOperations {
             queries: NativeQueries::empty(),
-            mutations: NativeQueries::empty(),
+            mutations: NativeMutations::empty(),
```

Comment:
we introduce NativeMutations because `NativeQueries` are now a map from `CollectionName`, and `NativeMutations` are a map from `ProcedureName`.